db/db_local/locations.go

```diff
@@ -48,9 +50,9 @@ func getDataLocation() string {
 	return loc
 }
 
-// location where the dump file will be stored, so that it can be later restored after connections
+// tar file where the dump file will be stored, so that it can be later restored after connections
```

Comment:
please can we update the other inconsistently named functions in this file

Reply:
other PR

Reply:
https://github.com/turbot/steampipe/issues/1936